gateway/modules/schema/helpers.go

```diff
@@ -35,11 +35,17 @@ func getSQLType(dbType, typename string) (string, error) {
 		return "float", nil
 	case model.TypeInteger:
 		return "bigint", nil
+	case model.TypeJsonb:
+		if dbType != string(utils.Postgres) {
+			return "", fmt.Errorf("jsonb not supported for database %s", dbType)
+		}
+		return "jsonb", nil
 	default:
 		return "", fmt.Errorf("%s type not allowed", typename)
 	}
 }
 
+// TODO ANY PARTICULAR DIRECTIVE THAT IS NOT SUPPORTED FOR JSONB
```

Comment:
What is this for?

Reply:
i have not tested jsob with any particular directive like adding @primary or @unique for a field type having json
so the question is does JSON supports all directives ?
if no then what directives are not supported
we can throw err while checking for error in schema creation

Reply:
Create an issue for this. Indexes on jsonb types

Reply:
ok

Reply:
so should i throw error now if the user tries to create index on jsonb types

Reply:
Yes